Common/ImageSheets/RectangleAllocator.cs

```diff
@@ -81,8 +81,9 @@ public static Chunk[] Allocate(IEnumerable<Image> images)
                     {
                         // move to the next row.
                         x = 0;
+                        var temp = topY;
                         topY = bottomY;
-                        bottomY = topY;
+                        bottomY = temp;

```

Comment:
This is actually wrong and flips the row upside down. I'm not sure how `bottomY = topY` came about (maybe an artifact of some refactoring that wasn't cleaned up), but they should both be pointing at the same Y pixel coordinate of the next image sheet row allocation, which is empty and so it currently has no height. 

Correct solution is to just remove `bottomY = topY;` outright.